src/js/components/SchemaField.js

```diff
@@ -277,6 +296,10 @@ class SchemaField extends Component {
           return this.renderSelect(errorMessage, autofocus, this.props);
         }
 
+        if (schema.media && schema.media.type === "placement") {
```

Comment:
The `type` is wrong. Please change to "application/x-**constraints**+json" as documented in the respective JIRA. I'd also change this to a regular expression as we eventually want to support different variations (for details please see JIRA).

Reply:
Yes, this is my bad in being misleading about this. I used "placement" just as my temporary data. The Jira with the proper types is here: https://jira.mesosphere.com/browse/INFINITY-2620

Reply:
That was actually my fault not copying the linked issue 😅